Add EBR-II models Refs #487 (#490)
idaholab/virtual_test_bed: doc/content/sfr/subchannel/EBR-II/EBR-2.md

| Inlet coolant temperature (K) | 624.7 | 616.4 |
| Outlet Pressure (kPa) | 200.0 | 200.0 |

During the SHRT transients, the mass flow rate and power vary. The normalized power and mass flow rate during the transients have been adapted from [!cite](mochizuki2014benchmark) , [!cite](mochizuki2018benchmark) and are presented in [fig:Norm_TR]. This information is used as input for the SCM transient calculations. The work in the cited sources utilizes a NETFLOW++ [!cite](mochizuki2010development) simulation to inform a COBRA-IV-I [!cite](wheeler1976cobra) model of the instrumented sub-assembly XX09. It should be noted that for SHRT-17, the power generation throughout the transient is solely due to decay heat because the protection system shuts down the reactor. For the SHRT-45R test, fission power continues into the transient for some time until the reactivity feedback mechanisms ultimately shut down the reactor.

!media subchannel/Normalized_Transients.png
    style=width:60%;margin-bottom:2%;margin:auto;
    id=fig:Norm_TR
    caption=Transient boundary conditions

## Steady State Results

Three simulation results are presented here and compared with experimental measurements:

> - item Results from the [DASSH](https://github.com/dassh-dev/examples/tree/master/Example-3) [!cite](atz2021ducted),subchannel code, which is used as a reference for subchannel agreement.

Reviewer: ```suggestion
- Results from the [DASSH](https://github.com/dassh-dev/examples/tree/master/Example-3) [!cite](atz2021ducted) subchannel code, which is used as a reference for verification.
```

> - item Results from SCM with a uniform (axially and radially) pin power profile.

Reviewer: ```suggestion
- Results from SCM with a uniform (axially and radially) pin power profile.
```

!media subchannel/XX09_TTC.png
    style=width:60%;margin-bottom:2%;margin:auto;
    id=fig:TTC17
    caption=Test SHRT-17

!media subchannel/XX09_TTC45.png
    style=width:60%;margin-bottom:2%;margin:auto;
    id=fig:TTC45
    caption=Test SHRT-45R

The above SCM results can be further improved by coupling to a Pronghorn model. This model can calculate the heat flux from the edge subchannels to the inner duct of the thimble, and apply this to the SCM simulations. This coupled simulation improves the SCM calculation specifically for the subchannels closest to the duct [!cite](tano2024validation).

## Transient Results

> Simulations a uniform (radial and axial) pin power profile are performed with the SCM model. The results are compared against experimental measurements and the NETFLOW++/COBRA simulations which are used as a reference result. The temperature transient calculations, together with the experimental measurements at thermocouple location TTC-31, are presented in [fig:transient1],[fig:transient2]. In SHRT-17, the power of the XX09 sub-assembly decreases rapidly at time zero due to the reactor scram, which results in a sudden coolant temperature decrease. On the other hand, in SHRT-45R there is no scram of the reactor, so power decreases at a much lower pace. This is why no sudden temperature drop is observed at time zero.

Reviewer: ```suggestion
Simulations with a uniform (radial and axial) pin power profile are performed with the SCM model. The results are compared against experimental measurements and the NETFLOW++/COBRA simulations which are used as a reference result. The temperature transient calculations, together with the experimental measurements at thermocouple location TTC-31, are presented in [fig:transient1],[fig:transient2]. In SHRT-17, the power of the XX09 sub-assembly decreases rapidly at time zero due to the reactor scram, which results in a sudden coolant temperature decrease. On the other hand, in SHRT-45R there is no scram of the reactor, so power decreases at a much lower pace. This is why no sudden temperature drop is observed at time zero.
```
